.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: 🚀 Feature request
+description: Suggest an idea for this project.
+labels: enhancement
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Thanks for taking a minute to file a feature for aiomysql!**
```

Comment:
"aiomysql" is mentioned instead of the actual project name. Here and probably in other forms.